src/asciidoc/docs/refs/glossary.adoc

```diff
@@ -0,0 +1,33 @@
+= Glossary
+:page-slug: glossary
+:page-description: Definition of terms
+// Settings:
+:experimental:
```

Comment:
What is using this setting?

Reply:
Resolving this, since you (Rachel) already added it to the global attributes for btn: and menu:.